driverconfig/isilon_v260_v122.json

```diff
@@ -116,6 +116,14 @@
         "DefaultValueForController": "0777",
         "DefaultValueForNode": ""
       },
+      {
```

Comment:
@rajkumar-palani If we need not support 1.22, we don't need this file

Reply:
@rajendraindukuri - we need to have K8s 1.22 version for VISA Support.